assets/js/base/components/form-token-field/style.scss

```diff
@@ -20,10 +20,20 @@
 		border-radius: 0;
 		box-shadow: none;
 		color: #000;
+		padding: $gap-smaller;
 		position: relative;
 
+		> .components-flex {
+			box-sizing: border-box;
+			padding-bottom: 0;
+			padding-top: 0;
+		}
+
+		.components-form-token-field__token-text,
 		input[type="text"].components-form-token-field__input {
-			min-height: 30px;
+			min-height: calc(31.5px - 0.5em);
```

Comment:
Out of curiosity, could you explain what problem is this solving? I tried removing this and breaking it in various ways, and I didn't manage. What is the `min-height` needed for? Also, if that's needed, wouldn't this kind of pixel perfect approach be somewhat fragile? Since `em` depend on the `font-size` it seems that you can get into a very small min height on larger font sizes.

But, again, as I don't really get why this is here, perhaps I'm getting it all wrong.

Reply:
Good catch, I think that was a left-over from a previous approach to fix it. I removed it in 65fd96ee313fb961d740194b5ae2deac559dd572.